sources/deny.toml

```
@@ -33,42 +37,33 @@ exceptions = [
 name = "encoding_rs"
 version = "0.8.30"
 expression = "(Apache-2.0 OR MIT) AND BSD-3-Clause"
-license-files = [
-    { path = "COPYRIGHT", hash = 0x39f8ad31 }
-]
+license-files = [{ path = "COPYRIGHT", hash = 0x39f8ad31 }]
 
 [[licenses.clarify]]
 name = "ring"
 expression = "MIT AND ISC AND OpenSSL"
-license-files = [
-    { path = "LICENSE", hash = 0xbd0eed23 },
-]
+license-files = [{ path = "LICENSE", hash = 0xbd0eed23 }]
 
 [[licenses.clarify]]
 name = "webpki"
 expression = "ISC"
-license-files = [
-    { path = "LICENSE", hash = 0x001c7e6c },
-]
+license-files = [{ path = "LICENSE", hash = 0x001c7e6c }]
 
 [bans]
 # Deny multiple versions or wildcard dependencies.
 multiple-versions = "deny"
 wildcards = "deny"
 
-deny = [
-    { name = "structopt" },
-    { name = "clap", wrappers = [ "cargo-readme" ] },
-]
+deny = [{ name = "structopt" }, { name = "clap", wrappers = ["cargo-readme"] }]
 
 skip = [
-    # several dependencies use an older version of toml
-    { name = "toml", version = "=0.5" },
-    # tungstenite uses a newer version of base64, but many of the
-    # first party crates are still in this version
-    { name = "base64", version = "=0.13" },
-    # AWS SDK for rust uses an older version of fastrand
-    { name = "fastrand", version = "=1.9" },
+    # sdk still uses 0.7.5 but everything in this repo now uses 0.8.2
```

"sdk" is ambiguous. Use "settings sdk" everywhere to be more clear.

```suggestion
    # settings sdk still uses 0.7.5 but everything in this repo now uses 0.8.2
```